static/js/tools/choropleth/choropleth.ts

```diff
@@ -31,20 +31,33 @@ window.onload = () => {
     React.createElement(MainPane),
     document.getElementById("main-pane")
   );
+  
+  // Redirect to basic url if none provide.
+  if (window.location.search == "") {
+    window.location.search = "statVar=Count_Person_Employed&perCapita=t&geoId=country/USA"
+  }
 
   // TODO:(iancostello) Refactor the url builder.
   // Build api call from request.
   var base_url = build_choropleth_url(
-      ['statVar', 'perCapita', 'geoId', 'level', 'mdom'], true)
-
+    ["statVar", "perCapita", "geoId", "level", "mdom"],
```

Comment:
this should probably just get you the maps, and the other api (/child/statvars) to get the display data. it doesn't make sense for maintainability to have both paths return stat var data.

Reply:
[Changed in previous PR]